tests/DCPS/WriteDataContainer/WriteDataContainerTest.cpp

```diff
@@ -350,6 +350,8 @@ int run_test(int argc, ACE_TCHAR *argv[])
 
   try
     {
+    DDS::DomainParticipantFactory_var dpf =
```

Comment:
OK, I see where it was happening implicitly before.  It would be nice to have a mock DomainParticipantImpl there, but that's for another PR.